.github/workflows/pr-build.yml

```diff
@@ -0,0 +1,98 @@
+name: "PR Build"
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  build:
+    name: build
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - id: setup-java-8
+        name: Setup Java 8
+        uses: actions/setup-java@v2
+        with:
+          distribution: adopt
+          java-version: 8
+      - uses: burrunan/gradle-cache-action@v1
+        name: restore gradle cache
```

Comment:
We don't need this step, the other ones use cache transparently without explicit restore

Reply:
Removed